index.html

```diff
@@ -223,14 +228,28 @@
         The term <dfn>webapp server</dfn> refers to server-side components of a <a>webapp</a>.
       </p>
       <p>
-        The term <dfn>push message</dfn> refers to an indication to a <a>webapp</a> that there is
-        new information for it from the <a>webapp server</a>, all or part of which MAY be contained
-        in the <a>push message</a> itself.
+        A <dfn>push message</dfn> is an indication to a <a>webapp</a> that there is new information
+        for it from the <a>webapp server</a>, all or part of which MAY be contained in the <a>push
+        message</a> itself.
+      </p>
+      <p>
+        A <dfn>push subscription</dfn> is a message delivery context that is established between
+        the <a>user agent</a> and the <a>push service</a> on behalf of a <a>service worker
+        registration</a>, and it is associated with that <a>service worker registration</a>. A
+        <a>push subscription</a> MUST NOT persist longer than its associated <a>service worker
+        registration</a>, but it MAY be terminated earlier.
```

Comment:
What about this:

A push subscription is a message delivery context established between the user agent and the push service on behalf of a webapp, and is associated with the webapp's service worker registration.

It avoids the double "service worker registration" term, s/it// and s/that is//.